javaslang/generator/Generator.scala

```diff
@@ -1558,6 +1569,15 @@ def generateTestClasses(): Unit = {
         """
       }
 
+      def genSimpleAliasTest(name: String, value: String): String = {
+        xs"""
+          @$test
+          public void should${name}ReturnNotNull() {
+              assertThat(${name.toLowerCase}($value)).isNotNull();
```

Comment:
this seemed fishy at first (the `toLowerCase`, instead of specifying it separately, like in `Either`), but it represents very well how a human expects it to work, good job) :)